PREREQUISITES.md

```diff
@@ -196,6 +196,19 @@ wget https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/libsgx-enclave-co
 sudo dpkg -i libsgx-enclave-common_2.3.101.46683-1_amd64.deb
 ```
 
+### Run aesm service on host machine
+If you are behind a corporate proxy,
+uncomment and update the proxy type and aesm proxy lines in /etc/aesmd.conf:  
+```
+proxy type = manual
+aesm proxy = http://your-proxy:your-port
+```
+
+Start aesm service on host machine  
```

Comment:
Can we put these lines in the beginning of this sub-section? The paragraph starting with the if is appearing as if starting the aesmd is an optional.

Reply:
I just put the two sections i.e setting proxy and running aesm to follow the sequence.

Reply:
Line starting with `###` would appear a sub-section. Under that I see 2 paragraphs, starting the `aesm` should be told first and then a note on machines behind the proxy network.